packages/validator/src/services/doppelgangerService.ts

```diff
@@ -118,11 +122,12 @@ export class DoppelgangerService {
       }
     }
 
-    this.logger.debug("doppelganger pollLiveness", {currentEpoch, indicesCount: indicesToCheckMap.size});
```

Comment:
This means we won't be logging anything now if there are no indices to check but I don't see how that's useful anyways